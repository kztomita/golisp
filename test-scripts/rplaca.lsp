(setq a '(a b c))

(print (rplaca '(a b) 'x))
(print (rplaca a (+ 1 2)))
(print (rplacd a '(4 5)))


