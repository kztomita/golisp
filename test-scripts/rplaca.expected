
(x b) 
(3 b c) 
(3 4 5) 
